example/docker-compose.yml

```
@@ -177,29 +190,29 @@ services:
   # C. REVERSE PROXY AND SSL SERVICES
   # --------------------------------------------------
 
-  # See https://github.com/nginx-proxy/nginx-proxy/blob/main/docs/README.md for setup and options
-  nginx-proxy:
-    image: nginxproxy/nginx-proxy
+  traefik:
+    image: traefik:v2.5
+    command:
+      - "--api.insecure=true"
```

```suggestion
      #- "--api.insecure=true"
```
I think this should be off by default.